docs/docker-testnet.md

```diff
@@ -19,8 +19,17 @@ To start a docker node and join the testnet run:
 docker run -d -p 3013:3013 aetrnty/epoch
 ```
 
-- *Your public IP address will be automatically determined and used in `peer_address` configuration option*
-- *Make sure you have a working port forwarding setup on your firewall to be able to fully participate in the testnet*
+Verify the node is running:
+```bash
+curl localhost:3013/v1/top
+```
+
+Your public IP address will be automatically determined and used in `peer_address` configuration option. You can change your `peer_address` by using `PEER_ADDRESS` Docker environment variable e.g.:
```

Comment:
Is this related to the peer_address in epoch.yaml ? Because in that peer_address the port can be specified as part of the address and in this example you only specify the IP address

Reply:
Only when reading the next commit, it is clear that EXTERNAL_PEER_ADDRESS is something different. Not sure why you need both.

Reply:
@ThomasArts Could you please check the latest changes, I changed this multiple times. The last we have is:
- EXTERNAL_PEER_ADDRESS (e.g. http://1.2.3.4:3013/)
- PEERS_ADDRESS_0 (e.g. http://31.13.248.102:3013/)

I really can't come up with better idea to provide and array/list of PEERS, thus this ugly PEERS_ADDRESS_0 and the cons. that we cannot set more than one peer address.